themes/default/content/blog/serverless-architecture-design/index.md

```diff
@@ -0,0 +1,116 @@
+---
+title: "Modern Serverless Architecture Design"
+date: 2023-11-03
+draft: false
+meta_desc: "Learn about the concept of serverless architecture and the different considerations to make when building out your serverless application."
+meta_image: meta.png
+authors:
+    - adora-nwodo
+tags:
+    - serverless
+    - serverless-functions
+    - system-design
+    - system-architecture
+
+---
+
+Serverless architectures have become popular because they make it easier for software developers to focus on application development without the additional operation overhead. With Pulumi, you can deploy and manage your serverless infrastructure and this article shows you different architectural design considerations for your serverless application.
```

Comment:
```suggestion
Serverless architectures are popular because they make it easier for software developers to focus on application development without additional operation overhead. With Pulumi, you can deploy and manage your serverless infrastructure, and this article shows you different architectural design considerations for your serverless application.
```

Its already a present fact.